### PR TITLE
kvm: mount existing volumes into vms

### PR DESCRIPTION
**Pull request by joanbe**
This commit fixes an issue where volumes already present on the libvirt
hypervisor were not mounted into newly created vms.

### CODE REVIEW

File: kvirt/config.py [hunk 1]
```diff
@@ -1338,15 +1339,18 @@ def plan(self, plan, ansible=False, url=None, path=None, autostart=False, contai
...
                 common.pprint("Disk %s skipped!" % disk, color='blue')
-                continue
+                disk_exists_on_client = True
             if len(vms) > 1:
                 shareable = True
             else:
                 shareable = False
-            newdisk = k.create_disk(disk, size=size, pool=pool, template=template, thin=False)
+            if disk_exists_on_client:
+                disk_to_add = k.disk_path(pool, disk)
```
Comment (karmab) [inline kvirt/config.py]: this would break on other platforms that libvirt since this method isnt defined elsewhere

File: kvirt/kvm/__init__.py [hunk 2]
```diff
@@ -159,6 +159,21 @@ def disk_exists(self, pool, name):
         except:
             return False
 
+    def disk_path(self, pool, name):
```
Comment (karmab) [inline kvirt/kvm/__init__.py]: cant we use disk_exists instead ?

File: kvirt/config.py (hunk 1 above)
Comment (joanbe) [reply kvirt/config.py]: This have been addressed in the latest commit: We now contition on self.type == 'kvm' as suggested.

File: kvirt/kvm/__init__.py (hunk 2 above)
Comment (joanbe) [reply kvirt/kvm/__init__.py]: As suggested on IRC, let disk_path function remain in the kvm package (could possibly be added to other providers later).

**Comment by karmab** [comment]
super sorry for taking so much time reviewing this one, summer didnt help i guess
anyway, i've implemented in a simpler way in https://github.com/karmab/kcli/commit/23dbe331f0d81f311c8a248fdde21e250f19cf86

**Pull request closed without merging**